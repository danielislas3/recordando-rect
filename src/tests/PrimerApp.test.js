import React from "react";
import "@testing-library/jest-dom";
// const { render } = require("@testing-library/react")
import { shallow } from "enzyme";
const { default: PrimerApp } = require("../PrimerApp");

describe("Pruebas en componente PrimeraApp", () => {
  // test('debe de mostrar el mensaje de Hola soy Daniel', () => {

  //   const saludo = 'Hola soy Daniel'

  //   const {getByText}= render(<PrimerApp saludo={saludo} />)

  //   expect(getByText(saludo)).toBeInTheDocument(saludo);
  // })

  test("debe mostror <PrimeraApp> correectamente ", () => {
    const saludo = "Hola soy Daniel";
    const wrapper = shallow(<PrimerApp saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });

  test("debe enviar el subtutulo enviado por props", () => {
    const saludo = "Hola soy Daniel";
    const subtitulo = "Soy un subtitulo con props";
    const wrapper = shallow(
      <PrimerApp saludo={saludo} subtitulo={subtitulo} />
    );
    const textoParrafo = wrapper.find("p").text();
    console.log(textoParrafo);
    expect(textoParrafo).toBe(subtitulo);
    
  });
});
