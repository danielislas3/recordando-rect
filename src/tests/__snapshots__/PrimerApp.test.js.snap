// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en componente PrimeraApp debe mostror <PrimeraApp> correectamente  1`] = `
<Fragment>
  <h1>
     
    Hola soy Daniel
  </h1>
  <p>
     
    Soy un subtitulo
  </p>
</Fragment>
`;
